examples/lusid/orders/blocks/4-set-new-property-to-block.sql

```diff
@@ -0,0 +1,34 @@
+-- define target blocks
```

Comment:
Apply the contingent order id to a set of blocks

Reply:
updated